### PR TITLE
Backend Client changes and documentation to simplify E2E demo steps

### PR DESCRIPTION
**Pull request by sawagh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,145 +14,91 @@ Under the covers matchmaking approaches touch on significant areas of computer s
...
-sudo apt-get update; sudo apt-get install -y -q python3 python3-virtualenv virtualenv make google-cloud-sdk git unzip tar
-# Setup your repository like Go workspace, https://golang.org/doc/code.html#Workspaces
-# This requirement will go away soon.
-mkdir -p $HOME/workspace/src/github.com/GoogleCloudPlatform/
-cd $HOME/workspace/src/github.com/GoogleCloudPlatform/
-export GOPATH=$HOME/workspace
-export GO111MODULE=on
-git clone https://github.com/GoogleCloudPlatform/open-match.git
-cd open-match
+# Install the example MMF and Evaluator.
```
Comment (Laremere) [inline README.md]: This line is wrong

File: docs/governance/templates/release_issue.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ and copy it into a release issue. Fill in relevent values, found inside {}
 {version} should be replaced with the version ie: 0.5.0.
```
Comment (Laremere) [inline docs/governance/templates/release_issue.md]: I think this file can be skipped.  I'm turning it into a issue template in #280

File: README.md (hunk 1 above)
Comment (sawagh) [reply README.md]: Removed it now - the explanation above seems sufficient.

File: docs/governance/templates/release_issue.md (hunk 2 above)
Comment (sawagh) [reply docs/governance/templates/release_issue.md]: Removed!

**Pull request merged**